js/data/locale/fa/AF/unitfmt.json

```diff
@@ -335,52 +356,59 @@
             "pound-force-foot": "#{n} پوند-فوت",
             "newton-meter": "#{n} نیوتن‌متر",
             "cubic-kilometer": "#{n}‎ km³",
-            "cubic-meter": "#{n} متر مکعب",
+            "cubic-meter": "#{n}m³",
             "cubic-centimeter": "#{n}‎ cm³",
-            "cubic-mile": "#{n} مایل مکعب",
-            "cubic-yard": "#{n} یارد مکعب",
-            "cubic-foot": "#{n} فوت مکعب",
-            "cubic-inch": "#{n} اینچ مکعب",
-            "megaliter": "#{n} مگالیتر",
-            "hectoliter": "#{n} هکتولیتر",
-            "liter": "#{n} لیتر",
-            "deciliter": "#{n} دسی‌لیتر",
-            "centiliter": "#{n} سانتی‌لیتر",
+            "cubic-mile": "#{n}mi³",
```

Comment:
note) Usually iLib prefer to provide their own language translation. but The latest CLDR changed unit translation to english

Reply:
I would not second-guess what the CLDR folks are doing. They changed it on purpose. Maybe that is the way it is actually represented in that language/country?

The only time we should override is when CLDR does not yet have translations, but ilib has ones previously provided by LG members. Those are for tg, mt, wo, and zu